src/background/service/preference.ts

```diff
@@ -4,7 +4,7 @@ import { keyringService, sessionService, i18n } from './index';
 import { TotalBalanceResponse, TokenItem } from './openapi';
 import { HARDWARE_KEYRING_TYPES } from 'consts';
 import { browser } from 'webextension-polyfill-ts';
-
+const version = process.env.version!.split('/')[0].replace(/[^0-9]/g, '');
```

Comment:
try `process.env.release`, no time string and extra text

Reply:
done